packages/@aws-cdk/aws-route53/test/util.test.ts

```diff
@@ -71,4 +72,24 @@ describe('util', () => {
     // THEN
     expect(qualified).toEqual('test.domain.com.');
   });
+
+  test('determineFullyQualifiedDomainName should not end with \'.\' when hostedZone.zoneName is an unresolved token', () => {
+    // GIVEN
+    const stack = new cdk.Stack();
+
+    // WHEN
+    const providedName = 'test';
+    new cdk.CfnOutput(stack, 'zoneNameExample', {
+      value: 'domain.com.',
+      exportName: 'zoneNameExample',
+    });
+
```

Comment:
Remove this empty line here, we don't want any empty lines unless they're between the given / when / then clauses